.github/workflows/test_definitions.yml

```diff
@@ -0,0 +1,111 @@
+name: Test Definitions
+on:
+  workflow_call:
+    inputs:
+      disable-vcs:
+        required: true
+        type: string
+      repository:
+        required: true
+        type: string
+        
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+jobs:
+  colcon-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout workspace
+        uses: actions/checkout@v3
+        with: 
+            repository: UBCSailbot/sailbot_workspace
+
+      - name: Checkout ROS package
+        if: ${{ inputs.repository }} != ${{ github.event.repository.name }}
+        uses: actions/checkout@v3
+        with: 
+            repository: UBCSailbot/${{ inputs.repository }}
+            path: src/${{ inputs.repository }}
+
+      - name: Test
+        uses: ./.github/actions/test/
+        env:
+          DISABLE_VCS: ${{ inputs.disable-vcs }}
```

Comment:
To resolve https://github.com/UBCSailbot/network_systems/pull/18#issuecomment-1474664342, add the `DISABLE_VCS` environment variable to the last step of the `ament-lint` and `clang-tidy` jobs, then rerun the workflows in your network systems PR. The error is related to VCS, but VCS should be disabled in subteam repository CI.

Reply:
Added 
```yml
        env:
          DISABLE_VCS: ${{ inputs.disable-vcs }}
 ```
 
to relevant places

Reply:
Resolved network systems security issue.